packages/tasks/src/model-libraries.ts

```diff
@@ -212,6 +212,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/cartesia-ai/cartesia_mlx",
 		snippets: snippets.cartesia_mlx,
 	},
+	"clip-scope": {
+		prettyLabel: "CLIP-ViT-L-scope",
+		repoName: "CLIP-ViT-L-scope",
+		repoUrl: "https://github.com/Lewington-pitsos/vitsae", 
```

Comment:
Hi @Lewington-pitsos, thanks for opening a PR :) Just to understand, what is the library here? For what I can see:
- `clip-scope` seems like a generic name that is not mentioned in both the model card and the github repo
- `clipscope` seems to be a Python library (_"from clipscope import ConfiguredViT, TopKSAE"_ in the model card) but can't see how to install it. At least it doesn't seem to be on PyPI.
- `CLIP-ViT-L-scope` is a very specific name (the name of the repo on the Hub) 
- `vitsae` is the name of the repo on the Hub. Is it planned to be a published library at some point? 

I'm asking this because depending on what you plan to do here, the integration with the Hub could change. Typically, do you think other repos might use the same library in the future or will it be a "one-shot"? We usually recommend to have a consistent naming between the library and repo tags to avoid confusion. Also, the name of the library is usually different from the name of the repo on the Hub, especially when multiple models are linked to the same library.